index.html

```diff
@@ -225,6 +225,52 @@ <h2 class="about-header-text">About me</h2>
         </article>
   </section>
   <!--end of the about myself section-->
+  <!--start of the contact form page-->
+  <section class="contact_form">
+    <h3 class="contact_mssg">
+      I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+    </h3>
+    <form  action="https://formspree.io/f/mrgvyylv" method="post" class="form">
+      <fieldset class="contact_imfo">
+        <label for="names">
+        <input 
+        type="text" 
+        class="field_box" 
+        name="names" 
+        placeholder="Full name"
+        required
+        maxlength="30">
+      </label><br>
+        <label for="mail">
+        <input 
+        type="email" 
+        class="field_box" 
+        name="email" 
+        placeholder="Email Address"
+        required
+        pattern=".*[.com]">
+      </label>
+        <textarea name="comment" id="comment_text" cols="30" rows="10" required maxlength="500" placeholder="write me something...."> 
+        </textarea>
```

Comment:
The placeholder for the textarea is not showing. Kindly check and ensure that it displays to help users know what they are suppose to use that for. :wink: